db/db_impl/db_impl_merge.cc

```diff
@@ -57,6 +57,32 @@ Status DBImpl::ValidateForMerge(const MergeInstanceOptions& mopts,
   return Status::OK();
 }
 
+Status DBImpl::CheckInRange(const Slice* begin, const Slice* end) {
+  Status s;
+  if (begin == nullptr && end == nullptr) {
+    return s;
+  }
```

Comment:
What does `nullptr` mean? Should we return `Status::InvalidArgument`?

Reply:
It means infinity. It's a common usage of nullptr within RocksDB codebase.